src/utils/counter.ts

```diff
@@ -1,11 +1,25 @@
 /**
  * Creates a counter object with initial value.
  * @param {number} [initialValue=0] - The initial value of the counter. Defaults to 0 if not provided.
+ * @param {string} [storageKey='counterPolling'] - The initial value of sessionStorage key. Defaults to counterPolling if not provided
  * @returns {Object} - Counter object with methods.
  */
-export const createCounter = (initialValue = 0) => {
+export const createCounter = (
+  initialValue = 0,
+  storageKey = "counterPolling"
+) => {
+  const savedValue = sessionStorage.getItem(storageKey);
```

Comment:
You can use `getSessionItem` but you have to define a costant key in `SessionItems`. It hepls you to serialize and deserialize values